SR2021/2021-01-01-january-update.md

```diff
@@ -0,0 +1,48 @@
+---
+to: SR2021 teams
+subject: Student Robotics January Update
+---
+
+Hello there!
+
+There is less than 2 weeks until our [first round of league matches](https://studentrobotics.org/events/sr2021/league-1/) so we wanted to check in and give you an update.
+
+## League Sessions
+
+The first league session will be happening on the 16th of January, at 12:00 UK time, lasting around 1.5 hours. You will need to have submitted your code to us (more on that below) by 09:00 UK time on the same day. If you do not submit code, or submit late, you will be marked absent and score 0 points.
```

Comment:
Have we worked out what time these are in Australia? Even if we're not including them in the email, it might be worth having it on the PR for reference.

Reply:
09:00Z is 19:30 in Adelade and 12:00Z is 22:30 in Adelade.

Reply:
Far from the best times, but unfortunately with 10.5h difference, it's the best we could do.